buttercup.el

```diff
@@ -814,39 +895,13 @@ Do not change the global value.")
 ;;;;;;;;;;;;;
 ;;; Reporters
 
-(defvar buttercup-reporter #'buttercup-reporter-adaptive
```

Comment:
What about letting the variable in the "Reporters" section and adding

```
(eval-when-compile
  ;; Defined below in a dedicated section
  (defvar buttercup-reporter))
```

before its first use?